dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/acl/DefaultTrackerOwnershipManager.java

```diff
@@ -154,15 +158,17 @@ public void grantTemporaryOwnership(
       throws ForbiddenException {
     validateGrantTemporaryOwnershipInputs(trackedEntity, program, user);
 
+    TrackedEntity hibernateTrackedEntity = manager.get(TrackedEntity.class, trackedEntity.getUid());
```

Comment:
Please add a `TODO(tracker) jdbc-hibernate: ....` so we mark all the places like this to revisit them. `jdbc-hibernate` is just an idea of a common prefix so we can search for `TODO(tracker) jdbc-hibernate`